### ISSUE TITLE
Handle Uphold capabilities for creators

### ISSUE THREAD

**Issue by Miyayes** [brave-intl/publishers#3642]
## Description

See **Uphold capabilities error handling** document.

**Comment by OAuthBringer** [comment]
I've verified the integration element for capabilities is functional in production. 

```
irb(main):007:0> p.uphold_connection.connection_client.users.get_capability("deposits")
I, [2022-07-01T11:51:37.129450 #92]  INFO -- request: GET https://api.uphold.com/v0/me/capabilities/deposits
I, [2022-07-01T11:51:37.326675 #92]  INFO -- response: Status 200
=> <Uphold::Types::UpholdUserCapability category="permissions", enabled=true, key="deposits", name="Deposits", requirements=[], restrictions=[]>
```

**Comment by OAuthBringer** [comment]
worked like a charm first time too. That never happens.

**Comment by OAuthBringer** [comment]
Again waiting on some feedback from Uphold.

**Comment by OAuthBringer** [comment]
Resolved by #3810
